gui/hud.gui

```diff
@@ -225,240 +225,258 @@ widget = {
+						{
 							vbox = {
-								text_label_center = {
-									text = "PF_HEIR"
+								layoutpolicy_horizontal = expanding
+								layoutpolicy_vertical = expanding
+								margin_top = 10
+								spacing = 15
+
+								background = {
+									texture = "gfx/interface/illustrations/event_scenes/wc_horde_vs_human.dds"
+									margin = { -5 0 }
+									fittype = end
+									alpha = 0.4
+									framesize = { 1200 848 }
+
+									modify_texture = {
+										texture = "gfx/interface/component_masks/mask_fade_vertical_up.dds"
+										blend_mode = alphamultiply
+									}
 								}
-								portrait_head = {
-									datacontext = "[GetPlayer.MakeScope.Var('gui_pol_faction_heir').GetCharacter]"
-									blockoverride "portrait_button"
-									{
-										using = tooltip_ne
+								
+								# LEADER
+								vbox = {
+									text_label_center = {
+										text = "CV_LEADER"
 									}
-									blockoverride "glow_visible"
-									{
-										visible = no
+									portrait_head = {
+										datacontext = "[GetPlayer.MakeScope.Var('gui_pol_faction_leader').GetCharacter]"
+										blockoverride "portrait_button"
+										{
+											using = tooltip_ne
+										}
+										blockoverride "glow_visible"
+										{
+											visible = no
+										}
 									}
 								}
-							}
-							vbox = {
-								text_label_center = {
-									text = "PF_VICE_LEADER"
-								}
-								portrait_head = {
-									datacontext = "[GetPlayer.MakeScope.Var('gui_pol_faction_vice_leader').GetCharacter]"
-									blockoverride "portrait_button"
-									{
-										using = tooltip_ne
+								
+								# HEIR AND VICE LEADER
+								hbox = {
+									spacing = 20
+									vbox = {
+										text_label_center = {
+											text = "PF_HEIR"
+										}
+										portrait_head = {
+											datacontext = "[GetPlayer.MakeScope.Var('gui_pol_faction_heir').GetCharacter]"
+											blockoverride "portrait_button"
+											{
+												using = tooltip_ne
+											}
+											blockoverride "glow_visible"
+											{
+												visible = no
+											}
+										}
 									}
-									blockoverride "glow_visible"
-									{
-										visible = no
+									vbox = {
+										text_label_center = {
+											text = "PF_VICE_LEADER"
+										}
+										portrait_head = {
+											datacontext = "[GetPlayer.MakeScope.Var('gui_pol_faction_vice_leader').GetCharacter]"
+											blockoverride "portrait_button"
+											{
+												using = tooltip_ne
+											}
+											blockoverride "glow_visible"
+											{
+												visible = no
+											}
+										}
 									}
 								}
-							}
-						}
-						
-						# DESC
-						vbox = {
-							layoutpolicy_horizontal = expanding
-							margin = { 10 5 }
-							spacing = 10
+								
+								# DESC
+								vbox = {
+									layoutpolicy_horizontal = expanding
+									margin = { 10 5 }
```

Comment:
To center the union title.
```suggestion
```
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/134358731-526282bd-aa1d-4ce7-90a6-02c9eb62d8b6.png)

</details>